src/main/java/org/jetbrains/bsp/bazel/server/bazel/data/ProcessResults.java

```diff
@@ -26,8 +29,8 @@ public InputStream getStderrStream() {
     return stderr;
   }
 
-  public int getExitCode() {
-    return exitCode;
+  public StatusCode getExitCode() {
```

Comment:
now the naming is inconsistent: getExitCode returns a StatusCode object.

Reply:
right